website/docs/concepts/recorder_service/README.md

```diff
@@ -159,10 +189,10 @@ For example:
 
 ## How to Use
 
-Data can be recorded through the Recorder App, or using the `EventsClient` class.
+Data can be recorded through the App App, or using the `EventsClient` class.
 
 To record data through the UI on the Brain display, see the
-[Recorder App Guide](/docs/apps/recorder_app/).
+[Launcher Status Bar](/docs/apps/launcher/#status-bar).
 
```

Comment:
Link to camera app

Reply:
Here: 356ab79d0af823a4e2c1306ff7e34c5943377ba6